PennMobile/src/main/java/com/pennapps/labs/pennmobile/MainActivity.java

```diff
@@ -116,7 +117,8 @@ public void onBackPressed() {
             mDrawerLayout.closeDrawer(mDrawerList);
             return;
         }
-
+//        super.onBackPressed();
+        // TODO slight bit of flickering when opening in browser (aka custom chrome tab)
```

Comment:
Never leave commented out code in the final release, if it's not needed just delete it